src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
+      PICS: S.S.A0000 && S.S.A0001 && S.S.A0002 && S.S.A0003
+      command: "readAttribute"
+      attribute: "SceneValid"
+      response:
+          value: true
+          constraints:
+              type: uint8
+
+    - label: "Reboot target device"
+      PICS: PICS_SDK_CI_ONLY
+      cluster: "SystemCommands"
+      endpoint: 0
+      command: "Reboot"
+
+    - label: "Reboot target device(DUT)"
+      verification: |
+          Did the DUT successfully reboot?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value: "Please reboot the DUT and enter 'y' after DUT starts"
+              - name: "expectedValue"
+                value: "y"
 
     - label:
           "TH configures AC2 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC2 which is different from AC1.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 200
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH sends a RecallScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x01."
       PICS: S.S.C05.Rsp
-      verification: |
-          If a status response is expected, DUT sends a response to TH with the Status field equal to 0x00 (SUCCESS). DUT returns to AC1.
-      disabled: true
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
 
     - label:
           "TH sends a ViewScene command to DUT with the GroupID field set to
           0x0001 and the SceneID field set to 0x01."
       PICS: S.S.C01.Rsp
-      verification: |
-          DUT sends a ViewSceneResponse command to TH with the Status field set to 0x00 (SUCCESS), the GroupID field set to 0x0001, the SceneID field set to 0x01, the TransitionTime field set to 0x0000 and a set of extension fields appropriate to AC1.
-      disabled: true
+      command: "ViewScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 0x0000
+              - name: "ExtensionFieldSets"
+                value:
```

Comment:
The spec does not require these things to be in this order....  And again, this only makes sense for the specific application we have here.